blog/2020-05-11-version-0.5.0-released.md

```diff
@@ -0,0 +1,86 @@
+---
+id: version-0.5.0-released
+title: Version 0.5.0 released
+author: Beanow
+author_url: https://github.com/Beanow
+author_image_url: https://avatars3.githubusercontent.com/u/497556?v=4&s=200
+---
+
+It's been months since the last versioned release, so there's a lot new to
+v0.5.0!
+
+You can find the release [on GitHub][GitHub release] or as a [Docker image].
+More details on how to use them can be found in the [installation guide].
+
+Since this is a large release, I'll only touch on several major additions here.
+
+### Discourse plugin
+
+Discourse forums can be analyzed with this release. And from using it so far, I
+think it's a really interesting plugin.
+
+Discourse forums by default require no authentication to be set up. Meaning you
+can load most Discourse forums out there and explore them with SourceCred, no
+approval required. The [plugin setup guide][discourse setup] walks through the
```

Comment:
This is a broken link for me

Reply:
Do you mean when testing it from GitHub?
From the preview: https://deploy-preview-75--priceless-morse-4119fb.netlify.app
And local testing this does work for me.